README.md

```diff
@@ -24,37 +24,37 @@ To run the integration tests with optional environment variables set, run:
 
     mvn -Poptional-env-vars verify
 
+You can also, optionally, supply `-Ddocker.showLogs` if you want to see the containers' logs as the integration tests are running.
+
 ## Running in Developer Mode
 
 To bring up the server locally for testing or development purposes, run:
 
-    mvn -Plive integration-test
```

Comment:
I think we should remove that profile if we're going to use the full stack in the container instead of vert.x alone(?)